Client/src/Utils/Theme/darkTheme.js

```diff
@@ -23,6 +23,9 @@ const {
 } = colors;
 
 const palette = {
+	action: {
+		disabled: border.dark.disabled,
+	},
```

Comment:
Is this action.disabled color being called anywhere? I didnt find it

Reply:
> Is this action.disabled color being called anywhere? I didnt find it

Possible I was playing around and forgot to remove it 🤔 I'll double check it as soon as I get a chance!

Reply:
Just had a look, this is required for the disabled borders for inputs